Jenkinsfile

```diff
@@ -121,6 +121,20 @@ pipeline {
                               build_jvm_linux()
                           }
                       }
+
+                      stage('build_osx_arm64') {
```

Comment:
Didn't we build the macosX64 binaries before? 🤔

Reply:
Ditto